### PR TITLE
Tidied up bind-dirs.md

### PR DESCRIPTION
**Pull request by unman**
Moved example to config-files.md

### CODE REVIEW

File: user/advanced-configuration/bind-dirs.md [hunk 1]
```diff
@@ -19,32 +19,38 @@ In a TemplateBasedVM all of the file system comes from the template except `/hom
...
 4. Save.
 
 5. Reboot the TemplateBasedVM.
 
 6. Done.
 
-If you added for example folder `/var/lib/tor` to the `binds` variable, from now on any files within that folder will persist reboots. If you added for example file `/etc/tor/torrc` to the `binds` variable, from now on any modifications to that file will persist reboots.
+From now on any files within the/var/lib/tor folder will persist across reboots.  
+
+You can make make many files or folders persist, simply by making multiple entries in the p50_user.conf file, each on a separate line.
```
Comment (marmarek) [inline user/advanced-configuration/bind-dirs.md]:
```suggestion
You can make make many files or folders persist, simply by making multiple entries in the 50_user.conf file, each on a separate line.  
```

File: user/advanced-configuration/bind-dirs.md [hunk 2]
```diff
@@ -54,35 +60,31 @@ If you added for example folder `/var/lib/tor` to the `binds` variable, from now
...
 ## How does it work? ##
 
-bind-dirs.sh is called on startup of a TemplateBasedVM, and configuration files in the configuration folders above are parsed to build a bash array.
+bind-dirs.sh is called at startup of a TemplateBasedVM, and configuration files in the above configuration folders are parsed to build a bash array.
 Files or folders identified in the array are copied to /rw/bind-dirs if they do not already exist there, and are then bind mounted over the original files/folders.
 
-Creation of the file and folders in /rw/bind-dirs should be automatic the first time the TemplateBasedVM is restarted after configuration.
+Creation of the files and folders in /rw/bind-dirs should be automatic the first time the TemplateBasedVM is restarted after configuration.
+
+If you want to circumvent this process, you can create the relevant filestructure under /rw/bind-dirs and make any changes at the same time that you perform the configuration, before reboot.
```
Comment (marmarek) [inline user/advanced-configuration/bind-dirs.md]:
```suggestion
If you want to circumvent this process, you can create the relevant file structure under /rw/bind-dirs and make any changes at the same time that you perform the configuration, before reboot.  
```

File: user/advanced-configuration/bind-dirs.md (hunk 1 above)
Comment (andrewdavidwong) [reply user/advanced-configuration/bind-dirs.md]: Fixed: https://github.com/QubesOS/qubes-doc/commit/e1248ad12022671f15a5912751967fd1846bfddd

File: user/advanced-configuration/bind-dirs.md (hunk 2 above)
Comment (andrewdavidwong) [reply user/advanced-configuration/bind-dirs.md]: Fixed: https://github.com/QubesOS/qubes-doc/commit/e1248ad12022671f15a5912751967fd1846bfddd

**Pull request merged**